### PR TITLE
docs: mark deprecated for date-picker defaultPickerValue prop

### LINKED ISSUE #41974: DatePicker - defaultPickerValue doesn't display
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/s/antd-reproduction-template-forked-uzk5ye?file=/index.js)

### Steps to reproduce

Click the date picker on the left and view the default picked value. 

### What is expected?

It should show the date/time that is passed in the `defaultPickerValue` prop.

### What is actually happening?

It isn't showing a default value. Nothing seems to happen. The value works correctly when using the `defaultValue` prop.

| Environment | Info |
| --- | --- |
| antd | 4.24.9 |
| React | 17.0.1 |
| System | Windows 10 |
| Browser | Chrome 112.0.5615.138 |

### PR DESCRIPTION
**Pull request by zbw-zbw**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close #41974

### 💡 Background and solution
- 需求背景：
- defaultPickerValue 已弃用，但文档并未更新描述
   image
   image
- 解决方案
    - 修改已弃用属性 defaultPickerValue 相关文档描述

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Modified the document of the deprecated property defaultPickerValue    |
| 🇨🇳 Chinese |     修改已弃用属性 defaultPickerValue 相关文档描述      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 40b9806</samp>

This pull request updates the documentation of the `DatePicker` and `RangePicker` components to warn users about the deprecation of the `defaultPickerValue` property. It advises users to use the `defaultValue` property instead, and informs them that the `defaultPickerValue` property will be removed in version 4.0.0.

### <samp>🤖 Generated by Copilot at 40b9806</samp>

*  Add deprecation warning to `defaultPickerValue` property of `DatePicker` and `RangePicker` components in English and Chinese documentation ([link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-278af64bde59169ca85bf2d7df8a110094a7c153d88b5277c1614c365e65b910L124-R124), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-278af64bde59169ca85bf2d7df8a110094a7c153d88b5277c1614c365e65b910L142-R142), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-278af64bde59169ca85bf2d7df8a110094a7c153d88b5277c1614c365e65b910L155-R155), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-278af64bde59169ca85bf2d7df8a110094a7c153d88b5277c1614c365e65b910L166-R166), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-278af64bde59169ca85bf2d7df8a110094a7c153d88b5277c1614c365e65b910L177-R177), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-278af64bde59169ca85bf2d7df8a110094a7c153d88b5277c1614c365e65b910L191-R191), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-4f6ca776b129678ab875461618596d96f83d09613799c0180d0489e7486f0cdeL125-R125), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-4f6ca776b129678ab875461618596d96f83d09613799c0180d0489e7486f0cdeL143-R143), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-4f6ca776b129678ab875461618596d96f83d09613799c0180d0489e7486f0cdeL156-R156), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-4f6ca776b129678ab875461618596d96f83d09613799c0180d0489e7486f0cdeL167-R167), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-4f6ca776b129678ab875461618596d96f83d09613799c0180d0489e7486f0cdeL178-R178), [link](https://github.com/ant-design/ant-design/pull/43781/files?diff=unified&w=0#diff-4f6ca776b129678ab875461618596d96f83d09613799c0180d0489e7486f0cdeL192-R192)) in `components/date-picker/index.en-US.md` and `components/date-picker/index.zh-CN.md`

### CODE REVIEW

File: components/date-picker/index.zh-CN.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ import locale from 'antd/locale/zh_CN';
 
 | 参数 | 说明 | 类型 | 默认值 | 版本 |
 | --- | --- | --- | --- | --- |
-| defaultPickerValue | 默认面板日期 | [dayjs](https://day.js.org/) | - |  |
+| defaultPickerValue | 默认面板日期（已弃用，请使用 `defaultValue` 代替） | [dayjs](https://day.js.org/) | - |  |
```
Comment (afc163) [inline components/date-picker/index.zh-CN.md:178]: 这两个感觉没办法互相代替。

Comment (zbw-zbw) [reply components/date-picker/index.zh-CN.md:178]: 感觉两者目的都是设置默认日期？目前 defaultPickerValue 已经弃用了 使用 defaultValue 可以实现设置默认日期的需求
image

image

**Review by AbdullahWins — APPROVED** [review]
good job!

**Comment by zbw-zbw** [comment]
> good job!

thank you

**Comment by zbw-zbw** [comment]
> good job!

how can merge this pr？

**Comment by zbw-zbw** [comment]
@afc163 大佬，看下这个 pr，rc-picker 那边已经 review 过了，是不是这边也没问题 可以合并了呢？
https://github.com/react-component/picker/pull/661
image

File: components/date-picker/index.zh-CN.md (hunk 1 above)
Comment (Wxh16144) [reply components/date-picker/index.zh-CN.md:178]: 抛弃偏右大佬的review，如果可以代替，弃用的props可以直接从文档中移除，控制台console警告就可以啦

Comment (zbw-zbw) [reply components/date-picker/index.zh-CN.md:183]: > 抛开偏右大佬的review，如果结论是可以代替，弃用的props可以直接从文档中移除，控制台console警告就可以啦

done!

**Review by zombieJ — APPROVED** [review]

**Comment by zombieJ** [comment]
CI failed. Pls check.

**Comment by zbw-zbw** [comment]
> CI failed. Pls check.

看起来像是 master 分支有点问题，应该不是我这边改动导致的 failed 🤔

**Comment by zbw-zbw** [comment]
> > CI failed. Pls check.
> 
> 看起来像是 master 分支有点问题，应该不是我这边改动导致的 failed 🤔

我看 master 分支的问题修复了 现在没问题了

**Pull request merged**